.idea/$CACHE_FILE$

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you shouldn't send `.idea` folder as it's for your IDE. So can you add it to the `.gitignore`?

Reply:
@abdullahceylan  I have added this to gitignore